### ISSUE TITLE
Unable to adjust the temperature for OpenAI Connector - currently set to 1

### ISSUE THREAD

**Issue by cheeksc** [microsoft/semantic-kernel#4672]
Unable to adjust the temperature for OpenAI Connector - currently set to 1.
This leads to inconsistent results when given the same prompt.

**Comment by markwallace-microsoft** [comment]
@cheeksc You can specify the temperature to use when invoking the prompt as follows:

```
        var result = await kernel.InvokePromptAsync(
            prompt,
            new(new OpenAIPromptExecutionSettings()
            {
                MaxTokens = 60,
                Temperature = 0.7
            }));
```

Here is a full example showing this: https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/KernelSyntaxExamples/Example58_ConfigureExecutionSettings.cs

You can also configure this in the prompt configuration, see: https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/KernelSyntaxExamples/Resources/GenerateStory.yaml

**Comment by cheeksc** [comment]
I should've specified that temperature adjustment isn't possible when using the Handlebars Planner

**Comment by gitri-ms** [comment]
@cheeksc I'm working on a fix for that (tracked by #4731)

**Comment by markwallace-microsoft** [comment]
Closing this as the issue is tracked elsewhere
